DS/Concordance.java

```diff
@@ -0,0 +1,45 @@
+package Assignments;
+
+/**
+ * Write a program which create a concordance of characters occurring in a string 
+(i.e which characters occur where in a string). 
+Read the string from the command line.
+Example:  java Concordance Hello World
+{d=[9], o=[4, 6], r=[7], W=[5], H=[0], l=[2, 3, 8], e=[1]}
+ */
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * @author Anjita 
+ * class concordance which create a concordance of characters occurring in a string.
+ */
+public class Concordance {
+	public static void main(String[] args) {
+		String str1[] = { "hello", "world" };
```

Comment:
DO not hard the value. Write proper unit test case and function to calculate the concordance.